### PR TITLE
Update Remove-AppxPackage.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #2082: Resolution.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/appx/Remove-AppxPackage.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ This command removes an app package named package1_1.0.0.0_neutral__8wekyb3d8bbw
 
 ### -AllUsers
 
-This cmdlet removes the app package for all user accounts on the computer. This cmdlet works off the parent package type. If it is a bundle, use -PackageTypeFilter and specify the bundle. To use this parameter, you must run the command by using administrator permissions.
+This cmdlet removes the app package for all user accounts on the computer. This cmdlet works off the parent package type. If it is a bundle, use -PackageTypeFilter with "Get-AppxPackage" command and specify the bundle. To use this parameter, you must run the command by using administrator permissions.
```
Comment (get-itips) [inline docset/windows/appx/Remove-AppxPackage.md:64]:
```suggestion
This parameter removes the app package for all user accounts on the computer. This parameter works off the parent package type. If it is a bundle, use -PackageTypeFilter with "Get-AppxPackage" command and specify the bundle. To use this parameter, you must run the command by using administrator permissions.

```

Comment (get-itips) [reply docset/windows/appx/Remove-AppxPackage.md:64]: The idea is to pipe the result from Get-AppxPackage into the Remove- ?

Comment (JohanFreelancer9) [inline docset/windows/appx/Remove-AppxPackage.md:64]:
```suggestion
This parameter removes the app package for all user accounts on the computer. The parameter works off the parent package type. If it is a bundle, use -PackageTypeFilter with "Get-AppxPackage" command and specify the bundle. To use this parameter, you must run the command by using administrator permissions.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Agree with @get-itips plus one more edit.

**Review by dariomws — APPROVED** [review]
I'm ok after you apply the updates suggested by @get-itips and @JohanFreelancer9.

Thanks!

**Comment by sowjanya365** [comment]
Thank you so much for your inputs @get-itips @dariomws @JohanFreelancer9 and @e0i! 

I've updated the document! Kindly review it and proceed with merging. Thank you! :)

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**